buildtest/cli/__init__.py

```diff
@@ -422,15 +422,15 @@ def history_menu(subparsers, parent_parser):
     """
 
     history_subcmd = subparsers.add_parser(
-        "history", aliases=["hy"], help="Query build history"
+        "history", aliases=["hy"], help="Query build history", parents=[parent_parser]
```

Comment:
can you remove this from `buildtest history --pager` instead just put this in `buildtest history list`

Reply:
@shahzebsiddiqui Okay, I thought we are moving up the `--pager` option (buildtest history --pager list) as mentioned in the issue. I'll change that.